### ISSUE TITLE
How to use NewTransportWithProxy?

### ISSUE THREAD

**Issue by estr0gen** [Danny-Dasilva/CycleTLS#333]
### Description

How do I use NewTransportWithProxy? I cannot seem to find any example of it. Thanks.

### Issue Type

Support

### Operating System

Linux

### Node Version

None

### Golang Version

None

**Comment by tim-belanenko** [comment]
Same problem :/

**Comment by Estr0gen** [comment]
> Usage example
> 
> https://github.com/Danny-Dasilva/CycleTLS/blob/a120dacc1abdac2b96529ad61c155210e538674b/cycletls/tests/integration/proxy_test.go#L14-L37

Yes, but that is only with the CycleTLS client, not the fhttp transport.

**Comment by tim-belanenko** [comment]
> > Usage example
> > https://github.com/Danny-Dasilva/CycleTLS/blob/a120dacc1abdac2b96529ad61c155210e538674b/cycletls/tests/integration/proxy_test.go#L14-L37
> 
> Yes, but that is only with the CycleTLS client, not the fhttp transport.

I have same problem :(

**Comment by akudnamor** [comment]
find a solution????????

**Comment by Tanjhand** [comment]
Ja3 := "771,4865-4866-4867-49195-49199-49196-49200-52393-52392-49171-49172-156-157-47-53,65037-45-16-0-13-43-18-11-65281-23-17513-5-51-10-35-27,25497-29-23-24,0"
		UserAgent := headers["user-agent"]
		dialer, err := cycletls.NewConnectDialer("http://"+args.IpSession, UserAgent)
		if err != nil {
			log.Println(err)
		}
		sess.Transport = cycletls.NewTransportWithProxy(Ja3, UserAgent, dialer)

**Comment by Tanjhand** [comment]
> NewConnectDialer

copy  C:\Users\Administrator\go\pkg\mod\github.com\!danny-!dasilva\!cycle!t!l!s\cycletls@v1.0.26\connect.go newConnectDialer  to
NewConnectDialer

**Comment by Danny-Dasilva** [comment]
something like this should work 

https://github.com/Danny-Dasilva/CycleTLS?tab=readme-ov-file#example-using-your-own-custom-httpclient
